### ISSUE TITLE
is it alright to delete a folder called this = .xdm-app-data ?

### ISSUE THREAD

**Issue by afat123** [subhra74/xdm#1093]
## Checklist

- [x] I'm asking a question
- [x] I've looked through the README and FAQ for similar questions
- [x] I've searched the bugtracker for similar questions including closed ones

## Question

WRITE QUESTION HERE

as the title suggest. 
is it okay to delete that folder?
cuz the size of that folder is big, and my storage is limited.

best regards
wahyu jayadi

**Comment by Istayq-Ahmmed** [comment]
This folder belongs to "Xtreme Download Manager(XDM)". If you don't have the app anymore you can delete it

**Comment by Guff05** [comment]
Yes it just means that you have xtreem download manager installed and it collects data over time
